src/lib/stitching/gravity/__tests__/stitching.test.ts

```diff
@@ -1066,6 +1080,28 @@ describe("gravity/stitching", () => {
           info: expect.anything(),
         })
       })
+
+      it("resolves the thumbnailImage field with a copy of the request context", async () => {
```

Comment:
oo nice spec

Reply:
Thanks! Negated assertions like on L1095 can be tricky, but I did observe it fail first, then go green with the actual change.